docs/can-ndjson-stream.md

```diff
@@ -0,0 +1,43 @@
+@module {function} can-ndjson-stream
+@parent can-ecosystem
+@package ../package.json
+
+@description can-ndjson-stream parses an ndjson stream into a stream of JavaScript objects.
+
+@signature `ndjsonStream(stream)`
```

Comment:
I will try to do this again, but for some reason the docObject would give me an error whenever I put code in the signature so I removed it. I'll try to just debug it and find the cause.